Algorithms/Heuristic_Search.cpp

```diff
@@ -0,0 +1,65 @@
+#include <bits/stdc++.h>
```

Comment:
Hi @Heet-Patel-5304 , Thanks for contributing to this repository.
We do have some heuristic based search in this folder. Requesting you to move the CPP code to the same folder
https://github.com/Kavya-24/Snippets/tree/master/Artificial%20Intelligence/Informed%20Search/A%20Star
Thanks & happy coding !

Reply:
The File has been successfully moved to destined location. Looking forward to addressing any improvements.